broadbean/__init__.py

```diff
@@ -5,6 +5,7 @@
 from ._version import __version__
 from .blueprint import BluePrint
 from .broadbean import PulseAtoms
+from .deprecate import deprecate
```

Comment:
perhaps?
```suggestion
from .deprecate import DeprecationWarning, deprecate
```

Reply:
Are we sure that we want the broadbean public module to be only one level. It seems to me that we should rather not modify the toplevel imports in this pr and do this in a subsequent pr